index.html

```diff
@@ -189,10 +189,10 @@ <h3 class="lets">LET’S CONNECT</h3>
       <h1>Contact me</h1>
       <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</P>
     </div>
-    <form action="post">
-      <input type="text" value="Ndonyi" placeholder="Your Name Here" id="input" required/>
+    <form action="https://formspree.io/f/xayzojej" method="post">
+      <input type="text" name="name" placeholder="Your Name Here" max-lenght="30" id="input" required/>
       <input type="email" value="mauricendonyi40@gmail.com" placeholder="Your Email Here" id="input" required/>
-      <textarea id="textpanel" cols="30" rows="9" placeholder="Write Your Message Here"></textarea> 
+      <textarea id="textpanel" cols="30" rows="9" maxlength="500" placeholder="Write Your Message Here"></textarea> 
```

Comment:
- [ ] [OPTIONAL] Great job adding `maxlength` attribute to the `textarea`, however I would highly recommend putting the required attribute on to the message field as well as it is part of the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) but was missed by the previous reviewer

> A user cannot submit the form if one or more of the fields is empty.